### PR TITLE
creating a module using only npm

### PR DESCRIPTION
**Pull request by enieber**

### CODE REVIEW

File: src/guide/05-using-rollup-with-npm.md [hunk 1]
```diff
@@ -2,4 +2,75 @@
...
+ //src/foo.js
+ 
+ export default 42;
+ ```
+
+To run, create one script in package.json
+
+```js
+"scripts": {
+    "build": "node_modules/rollup/bin/rollup src/main.js --output bundle.js"
```
Comment (TrySound) [inline src/guide/05-using-rollup-with-npm.md]: Why `node_modules/rollup/bin/`? npm can resolve local bin. And your way won't work on windows.

**Comment by enieber** [comment]
because the node_modules/rollup/bin/rollup  will run the rollup locally. This is only one example of how to use the rollup using npm and i don't know if it works with Windows, but i know it work in Unix, so this is one start.

**Comment by TrySound** [comment]
@enieber 
> npm can resolve local bin

**Comment by Rich-Harris** [comment]
Yep, this is one of the great things about npm run scripts – it puts `node_modules/.bin` in `$PATH` before running the command, so 

```js
"build": "node_modules/rollup/bin/rollup src/main.js --output"
```

is equivalent to

```js
"build": "rollup src/main.js --output"
```

(because npm creates the relevant symlinks on installation). Would be awesome if you could just make that small change, then we can merge this in. Thanks!

**Comment by enieber** [comment]
Sorry I did not understand right, i already fix

**Comment by Rich-Harris** [comment]
:tada:

**Pull request merged**